velox/exec/tests/utils/LocalExchangeSource.cpp

```diff
@@ -130,11 +140,11 @@ class LocalExchangeSource : public exec::ExchangeSource {
           }
 
           if (!requestPromise.isFulfilled()) {
-            requestPromise.setValue();
+            requestPromise.setValue(Response{totalBytes, atEnd_});
           }
         });
 
-    return std::move(future);
+    return future;
```

Comment:
Why not moving anymore?

Reply:
The compiler now does this automatically and actually complains if I keep the move.